Trident/offsetfinder.c

```diff
@@ -16,1104 +16,1161 @@
 t_target_environment target_environment = NotSupported;
 
 t_target_environment info_to_target_environment(const char *device_model, const char *system_version) {
-    determineTarget("iPhone4,1", "9.2", iPhone41_iOS920);
-    determineTarget("iPhone4,1", "9.2.1", iPhone41_iOS921);
-    determineTarget("iPhone4,1", "9.3", iPhone41_iOS930);
-    determineTarget("iPhone4,1", "9.3.1", iPhone41_iOS931);
-    determineTarget("iPhone4,1", "9.3.2", iPhone41_iOS932);
-    determineTarget("iPhone4,1", "9.3.3", iPhone41_iOS933);
-    determineTarget("iPhone4,1", "9.3.4", iPhone41_iOS934);
-    determineTarget("iPhone5,2", "9.2", iPhone52_iOS920);
-    determineTarget("iPhone5,2", "9.2.1", iPhone52_iOS921);
-    determineTarget("iPhone5,2", "9.3.2", iPhone52_iOS932);
-    determineTarget("iPhone5,3", "9.2.1", iPhone53_iOS921);
-    determineTarget("iPhone5,3", "9.3.2", iPhone53_iOS932);
-    determineTarget("iPhone5,3", "9.3.3", iPhone53_iOS933);
-    determineTarget("iPhone5,4", "9.3.2", iPhone54_iOS932);
-    determineTarget("iPad2,1", "9.1", iPad21_iOS910);
+	// iPhone 4S (iPhone4,1)
```

Comment:
@EGYbkgo9449 could you please only commit specific changes for iPhone5,3 iOS 9.3.2? And also check the indentation, should be 4 spaces